src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -158,10 +172,14 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
         }
 
         this.setState({
-            isLoading: false,
+            isLoadingAccessStats: false,
             accessStats: undefined,
             accessStatsError,
         });
+
+        this.props.onError(error, ERROR_CODE_FETCH_ACCESS_STATS, {
```

Comment:
This is addressed as part of https://github.com/box/box-ui-elements/pull/661/files so you can remove, thanks for adding it though :)

Reply:
DONE!